api/v1beta1/hyperconverged_types.go

```diff
@@ -407,6 +407,19 @@ type HyperConvergedFeatureGates struct {
 	// +kubebuilder:default=false
 	// +default=false
 	EnableManagedTenantQuota *bool `json:"enableManagedTenantQuota,omitempty"`
+
+	// TODO update description to also include cpu limits as well, after 4.14
+
+	// Enable KubeVirt to set automatic limits when they are needed.
+	// If ResourceQuota with set memory limits is associated with a namespace, each pod in that namespace must have memory limits set.
+	// By default, KubeVirt does not set such limits to the virt-launcher pod.
+	// When this feature gate is enabled, KubeVirt will set limits to the virt-launcher pod if they are not set manually
+	// and if a resource quota with memory limits is associated with the creation namespace.
+	// Note: this feature is in Developer Preview.
+	// +optional
+	// +kubebuilder:default=false
```

Comment:
I'd expect this also on lines 69 and 704.

Reply:
@tiraboschi Can I ask you why the `persistentReservation` is not there?

Reply:
Good point, this smells like a bug but our tests are not detecting it.
So the tests are not robust enough or kube builder become smarter and duplicating it is not needed anymore.
Let's investigate.